server/sonar-web/src/main/js/apps/projects/components/ProjectCardLeak.js

```diff
@@ -44,8 +45,9 @@ type Props = {
     visibility?: boolean
   }
 };
+*/
 
-export default function ProjectCardLeak({ measures, organization, project }: Props) {
+export default function ProjectCardLeak({ measures, organization, project } /*: Props */) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this function to reduce its Cognitive Complexity from 18 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3776)